### PR TITLE
September 24 2024 - Creating Shivay Lamba spotlight

### PR DESCRIPTION
**Pull request by kmartens27**
This PR is resubmitting the [original page](https://github.com/jenkins-infra/contributor-spotlight/pull/192) to fix the build as it was not processing properly. This also adds in the edits from the docs review so that everything has been incorporated.

### CODE REVIEW

**Review by alyssat — DISMISSED** [review]
LGTM. Great read! Thanks @kmartens27 & @shivaylamba

**Comment by gounthar** [comment]
It is still failing, but I can't access the logs. 🤔

**Comment by kmartens27** [comment]
@gounthar I had some issues trying to get this to run on my local clone. After running `npm install eslint@latest --save-dev` I was able to get it to load and render properly. This updated my package-lock.json & package.json files with updated versions of eslint, among other things, so I was going to submit an issue to try and get that sorted. I didn't send all those changes with my PR because I didn't want to break anything or bring along other things here.

**Review by gounthar — APPROVED** [review]

File: src/pages/contributors/shivay-lamba.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Jenkins is indeed the number one tool for CI/CD automation, and it helped me from my first internship tasks in 2018 to managing an entire tech team in my first ever full-time role.
+The ability to integrate Jenkins reduces the time required to set up automation.
+
+In addition, Jenkins has also helped automate the deployment of Jupyter notebooks, the flagship project for Google Summer of Code 2020 with the Jenkins Data Science plugin.
+This project shows the capability and extensibility of being able to automate builds and testing for different types of applications. 
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+I am really passionate about how easy Jenkins makes it for DevOps engineers and engineering teams to make it super scalable for different environments.
+The ease of use, great developer experience, and community support, and open-source environment for Jenkins is unmatched, making it the best choice for self-deployment of CI/CD and build automation.
```
Comment (alyssat) [inline src/pages/contributors/shivay-lamba.adoc:62]:
```suggestion
The ease of use, great developer experience, community support, and open-source environment for Jenkins is unmatched, making it the best choice for self-deployment of CI/CD and build automation. 
```

**Review by alyssat — APPROVED** [review]
Just 1 very minor edit. Everything else LGTM.

**Pull request merged**